test_scripts/Resumption/Handling_errors_from_HMI/002_successful_resumption_WARNINGS_resultCode.lua

```diff
@@ -0,0 +1,91 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Requirement summary:TBD
+--
+-- Description:
+-- In case:
+-- 1. Rpc_n for resumptions is added by app
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App reregisters with actual HashId
+-- 4. Rpc_n request is sent from SDL to HMI during resumption
+-- 5. HMI responds with WARNINGS resultCode to Rpc_n request
+-- SDL does:
+-- 1. process WARNINGS response from HMI
+-- 2. restore persistent data
+-- 3. respond RegisterAppInterfaceResponse(success=true,result_code=SUCCESS) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Function ]]
+function common.sendResponse(pData, pErrorRespInterface, pCurrentInterface)
+  if pErrorRespInterface ~= nil and pErrorRespInterface == pCurrentInterface then
+    common.getHMIConnection():SendResponse(pData.id, pData.method, "WARNINGS", {})
+  else
+    common.getHMIConnection():SendResponse(pData.id, pData.method, "SUCCESS", {})
+  end
+end
+
+local function checkResumptionData(pAppId)
+  common.addSubMenuResumption(pAppId)
+  common.setGlobalPropertiesResumption(pAppId)
+  common.subscribeVehicleDataResumption(pAppId)
+  common.subscribeWayPointsResumption(pAppId)
+  common.getHMIConnection():ExpectRequest("UI.AddCommand",
+    {common.rpcs.addCommand.UI})
+  :Do(function(_, data)
+      common.sendResponse(data)
+    end)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand",
+    {common.rpcs.addCommand.VR},
```

Comment:
@GetmanetsIrina It seems here and below `common.resumptionData[pAppId]` has to be used instead of `common.rpcs`